bin/tictactoe

```diff
@@ -1,3 +1,21 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Welcome to Tic Tac Toe!"
+puts "How many players: 0, 1, or 2?"
+players = gets.strip.to_i
+if players == 0
+  player_1 = Player::Computer.new("X")
+  player_2 = Player::Computer.new("O")
+elsif players == 1
+  player_1 = Player::Human.new("X")
+  player_2 = Player::Computer.new("O")
+else players == 2
+  player_1 = Player::Human.new("X")
+  player_2 = Player::Human.new("O")
+end
+game = Game.new(player_1, player_2)
+until game.won?
+  game.play
+end
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.